libs/SmartSync/SmartSync/Classes/Manager/SFSmartSyncSyncManager.m

```diff
@@ -206,9 +213,9 @@ - (void) runSync:(SFSyncState*) sync updateBlock:(SFSyncSyncManagerUpdateBlock)u
     };
     
     // Run on background thread
+    updateSync(kSFSyncStateStatusRunning, 0, kSyncManagerUnchanged, kSyncManagerUnchanged);
```

Comment:
Updating from new to running before send the work to a background thread, otherwise the first update would say "new" (when all hybrid tests are expecting "running" - it was only working before because sync was passed by reference).